next-client/src/app/about/page.tsx

```diff
@@ -13,47 +13,399 @@ export default function AboutPage() {
+        <li>
+          <a href="https://ai.objectives.institute/blog/using-ai-to-give-people-a-voice-a-case-study-in-michigan">
+            Using AI to Give People a Voice, a Case Study in Michigan
+          </a>
+        </li>
+      </ul>
+      <br />
+
+      <h3 id="faq">FAQ</h3>
+
+      <h5>What inputs do I need to create a T3C report?</h5>
+      <ol className="list-decimal list-inside pl-2">
+        <li>A correctly formatted CSV file</li>
+        <li>An OpenAI API key</li>
+      </ol>
+      <br />
+
+      <h5>How do I sign into T3C?</h5>
+      <p>We currently allow signing in with accounts managed by Google:</p>
+      <ol className="list-decimal list-inside pl-2">
+        <li>Click the "Sign in" button in the top right corner</li>
+        <li>
+          Choose the Google account you want to sign in with and complete
+          authentication
+        </li>
+        <li>You'll then have access to your T3C dashboard</li>
+      </ol>
+      <br />
+
+      <h5>How secure is the data I send through T3C?</h5>
+      <p>
+        See our{" "}
+        <a className="underline" href="#privacy-security">
+          Privacy and security
+        </a>{" "}
+        section for detailed information on our policies.
+      </p>
+      <br />
+
+      <h5>Who can I contact for support?</h5>
+      <p>
+        Reach out to <a href="mailto:hello@objective.is">hello@objective.is</a>{" "}
+        with questions or feedback.
+      </p>
+      <br />
+
+      <h3 id="tutorial">Report creation FAQ</h3>
+
+      <h5>How do I create a new report?</h5>
+      <ol className="list-decimal list-inside pl-2">
+        <li>Click the "Create a report" button in the upper right corner</li>
+        <li>
+          Fill in required fields:
+          <ul className="list-disc list-inside pl-2">
+            <li>Report name (e.g. "Sentiment Analysis June 2025")</li>
+            <li>Report description (including context and purpose)</li>
+            <li>The dataset to analyze, as a CSV</li>
+            <li>Your OpenAI API key</li>
+          </ul>
+        </li>
+        <li>
+          Optionally, modify the prompts for any stage of the processing
+          pipeline to better fit your use case/data.
+        </li>
+      </ol>
+      <br />
+
+      <h5>What data format does the CSV file need to follow?</h5>
+      <ul className="list-disc list-inside pl-2">
+        <li>
+          Required column: "comment" (contains the text of the responses to
+          analyze with T3C)
+        </li>
+        <li>
+          Optional column: "interview" (respondent name, can be "Anonymous" or
+          blank)
+        </li>
+        <li>
+          Optional column: “id” (for linking to any external data, will be
+          generated if blank)
+        </li>
+      </ul>
+      <p>
+        A{" "}
+        <a
+          className="underline"
+          href="https://docs.google.com/spreadsheets/d/15cKedZ-AYPWMJoVFJY6ge9jUEnx1Hu9MHnhQ_E_Z4FA/edit?gid=0#gid=0"
```

Comment:
nit: what's that magic you mentioned to open this in a new window? target="_blank" ?